dts/arm/st/stm32f0.dtsi

```diff
@@ -92,6 +92,16 @@
 			status = "disabled";
 			label= "I2C_2";
 		};
+
+		spi1: spi@40013000 {
+			compatible = "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013000 0x400>;
+			interrupts = <25 5>;
+			status = "disabled";
+			label= "SPI_1";
```

Comment:
Add a space right before the = in all files changed:
label= "SPI_1";
to
label = "SPI_1";

Reply:
Fixed.